src/Microsoft.ApplicationInsights/DataContracts/DependencyTelemetry.cs

```diff
@@ -104,6 +105,12 @@ private DependencyTelemetry(DependencyTelemetry source)
         /// <inheritdoc />
         string IAiSerializableTelemetry.BaseType => nameof(RemoteDependencyData);
 
+        EventKeywords ISupportRichPayloadEventSource.EventSourceKeyword => RichPayloadEventSource.Keywords.Dependencies;
+
+        object ISupportRichPayloadEventSource.Data => this.Data;
```

Comment:
This is not correct.. It should be this.InternalData

Reply:
(in this case .Data is actually the field to store sqlcommand etc) name is confusing.